fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSExternalCatalog.java

```diff
@@ -203,6 +206,26 @@ protected void initLocalObjectsImpl() {
         icebergHiveCatalog = IcebergUtils.createIcebergHiveCatalog(this, getName());
     }
 
+    private void initHiveConfIfNeeded(HiveConf hiveConf) {
+        String resourcesPath = catalogProperty.getOrDefault(HMSProperties.HIVE_CONFIG_RESOURCES, null);
+        if (Strings.isNullOrEmpty(resourcesPath)) {
+            return;
+        }
+        for (String resource : resourcesPath.split(",")) {
+            // Construct the full path to the resource
+            String resourcePath = Config.external_catalog_config_dir + File.separator + resource.trim();
+            File file = new File(resourcePath); // Create a File object for the resource path
+
+            // Check if the file exists and is a regular file
+            if (file.exists() && file.isFile()) {
+                hiveConf.addResource(new Path(file.toURI())); // Add the resource to the configuration
+            } else {
+                // Handle the case where the file does not exist
+                LOG.warn("Hive configuration resource file {} does not exist", resourcePath);
```

Comment:
1. add catalog name in log.
2. What if xml does not found? Is there any impact?